### PR TITLE
Adventure

### PR DESCRIPTION
**Pull request by OmNomCodes**
HELLO DAAAAAD

### CODE REVIEW

File: adventure.py [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    answer = input(">")
+
+    if answer == "1":
+        print("Good idea, this will be a good investment.")
+
+    elif answer == "2":
+        money -= 50
+        print(" You decided to buy the cheapest hoverboard you can find.")
+        print("You buy a hoverboard for 50 microbucks.  It says it can hold 2 people.")
+        print("You have " + str(money) + " microbucks left.")
```
Comment (tedsiy) [inline adventure.py]: you should use fstrings

Comment (tedsiy) [reply adventure.py]:
```suggestion
        print(f"You have {money} microbucks left.")
```

**Review by tedsiy — CHANGES_REQUESTED** [review]
replace string concats with fstrings

File: adventure.py (hunk 1 above)
Comment (OmNomCodes) [reply adventure.py]: Good idea I have no idea how to do that but I do now.

**Comment by OmNomCodes** [comment]
Dad is there a way to make like a 'backpack' so I could store things in it, like say they buy a phone, then the phone would be in their backpack.  Just like the money, but a list of things.  Maybe an array or something?

**Comment by tedsiy** [comment]
> Dad is there a way to make like a 'backpack' so I could store things in it, like say they buy a phone, then the phone would be in their backpack. Just like the money, but a list of things. Maybe an array or something?

you can use a dictionary if you plan to have other items in the backpack you need to track. You can also modify the dictionary based on what you want to remember.
see https://www.w3schools.com/python/python_dictionaries.asp

**Review by tedsiy — APPROVED** [review]

**Pull request merged**